src/index.ts

```diff
@@ -20,16 +20,21 @@ export const formatValidationError = (error: t.ValidationError) => {
         error.context as Array<t.ContextEntry>
     );
 
-    return pipe(maybeErrorContext, map(errorContext => {
-        const expectedType = errorContext.type.name;
-        return (
-            // https://github.com/elm-lang/core/blob/18c9e84e975ed22649888bfad15d1efdb0128ab2/src/Native/Json.js#L199
-            // tslint:disable-next-line:prefer-template
-            `Expecting ${expectedType}`
-                + (path === '' ? '' : ` at ${path}`)
-                + ` but instead got: ${jsToString(error.value)}.`
-        );
-    }))
+    return pipe(
+        maybeErrorContext,
+        map((errorContext) => {
+            const expectedType = errorContext.type.name;
+            return (
+                // https://github.com/elm-lang/core/blob/18c9e84e975ed22649888bfad15d1efdb0128ab2/src/Native/Json.js#L199
+                // tslint:disable-next-line:prefer-template
+                `Expecting ${expectedType}` +
+                (path === '' ? '' : ` at ${path}`) +
+                ` but instead got: ${jsToString(error.value)}${
+                error.message ? ` (message: ${jsToString(error.message)})` : ''
```

Comment:
I'm not so sure about the `(message: ....)` format, what about just having the message in parens? Wdyt?

```suggestion
                error.message ? ` (${jsToString(error.message)})` : ''
```

Also, the formatting here inside template literal is getting a bit crazy. Maybe this is worth extracting.

Reply:
I'm OK either way! I'll remove it and look into simplifying the template.

Reply:
personally though I don't really mind the template string haha